plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/aps/system/services/mapping/SeoMappingDAO.java

```diff
@@ -138,24 +151,17 @@ protected void addRecord(FriendlyCodeVO vo, Connection conn) throws ApsSystemExc
 			closeDaoResources(null, stat);
 		}
 	}
-	
-	@Override
-	public void deleteMapping(String pageCode, String contentId) {
-		Connection conn = null;
-		try {
-			conn = this.getConnection();
-			conn.setAutoCommit(false);
-			this.deleteRecord(null, pageCode, contentId, conn);
-			conn.commit();
-		} catch (Throwable t) {
-			this.executeRollback(conn);
-			_logger.error("Error deleting apping",  t);
-			throw new RuntimeException("Error deleting apping", t);
-		} finally {
-			this.closeConnection(conn);
-		}
-	}
-	
+    
+    @Override
+    public void deleteMappingForContent(String contentId) {
+        super.executeQueryWithoutResultset(DELETE_FROM_CONTENTID, contentId);
+    }
+
+    @Override
+    public void deleteMappingForPage(String pageCode) {
+        super.executeQueryWithoutResultset(DELETE_FROM_PAGECODE, pageCode);
+    }
+/*
```

Comment:
Can we remove this?

Reply:
No because this method is called when the friendly code is removed from a page. There is a similar method for contents.

Reply:
I meant the commented out code